vignettes/miaTime.Rmd

```diff
@@ -215,7 +230,7 @@ BiocManager::install("mia")
 ```
 
 
-```{r, eval=FALSE}
+```{r eval=FALSE, include=FALSE}
```

Comment:
could we have eval=TRUE here and elsewhere? would be better for constant testing, if possible

Reply:
That was because I thought it would give error due to mia dependency. It is removed now, ready to be tested.